src/main/antora/modules/platform/pages/packaging/packaging001.adoc

```diff
@@ -39,5 +39,5 @@ Web modules are packaged as JAR files with a `.war` (web archive) extension.
 Application client modules are packaged as JAR files with a `.jar` extension.
 
 * Resource adapter modules, which contain all Java interfaces, classes, native libraries, and, optionally, a resource adapter deployment descriptor.
-Together, these implement the Connector architecture (see <<jakarta-connectors>>) for a particular EIS.
+Together, these implement the Connector architecture (see xref:intro:overview/overview.adoc#_jakarta_connectors[Jakarta Connectors]) for a particular EIS.
```

Comment:
This link is broken.

Reply:
The intro part hasn't been converted yet - there's just a placeholder there now so the target doesn't yet exist in git.

It does exist on my desktop however, I did a preliminary conversion of the intro and this link does seem to work as intended.